src/Document.tsx

```diff
@@ -58,47 +66,59 @@ const Document = (props: DocProps) => {
-                {doc?.state?.content ? (
-                  Object.entries(doc?.state?.content).map((entry: any[]) => (
-                    <Box mb={3}>
-                      <Text mb={1} fontWeight="bold">
-                        {entry[0] && entry[0].toString()}
-                      </Text>
-                      {typeof entry[1] === 'string' ? (
-                        <Text>{entry[1]}</Text>
-                      ) : (
-                        <Box
-                          backgroundColor={
-                            colorMode === 'dark' ? 'gray.900' : 'gray.100'
-                          }
-                          p={3}
-                          borderRadius={5}
-                        >
-                          <Code
-                            fontSize="sm"
-                            background="transparent"
-                            overflowX="auto"
-                            whiteSpace="pre"
-                            display="block"
+                {(docContent !== undefined) ? (
+                  (doc?.state.doctype === 'tile' &&
+                    Object.entries(docContent!).map((entry: any[]) => (
+                      <Box mb={3}>
+                        <Text mb={1} fontWeight="bold">
+                          {entry[0] && entry[0].toString()}
+                        </Text>
+                        {typeof entry[1] === 'string' ? (
+                          <Text>{entry[1]}</Text>
+                        ) : (
+                          <Box
+                            backgroundColor={
+                              colorMode === 'dark' ? 'gray.900' : 'gray.100'
+                            }
+                            p={3}
+                            borderRadius={5}
                           >
-                            {JSON.stringify(entry[1], undefined, 2)}
-                          </Code>
-                        </Box>
-                      )}
-                    </Box>
-                  ))
+                            <Code
+                              fontSize="sm"
+                              background="transparent"
+                              overflowX="auto"
+                              whiteSpace="pre"
+                              display="block"
+                            >
+                              {JSON.stringify(entry[1], undefined, 2)}
+                            </Code>
+                          </Box>
+                        )}
+                      </Box>
+                    )))
+                  || (doc?.state.doctype === 'caip10-link' &&
+                    <Text>{docContent.toString()}</Text>)
                 ) : (
-                  <>
+                  <Text>
                     {isLoading ? (
                       <Skeleton height="20px" width={400} />
                     ) : (
                       <Text>Get a document to see its content</Text>
                     )}
-                  </>
+                  </Text>
```

Comment:
The only I think might be worth adding now is a little note to say "No content" if the content object is empty. Because there are so many docs with no content. But not a blocker for merging this PR. I'll add an issue.